docs/devel/pull-requests.md

```diff
@@ -73,12 +74,34 @@ The following will save time for both you and your reviewer:
 
 ## Release Notes
 
-1. Your PR title is the **release note** you want published at release time.
-1. Release note labels are only needed on master branch PRs.
+This section applies only to pull requests on the master branch.
+
 1. All pull requests are initiated with a `release-note-label-needed` label.
 1. For a PR to be ready to merge, the `release-note-label-needed` label must be removed and one of the other `release-note-*` labels must be added.
 1. `release-note-none` is a valid option if the PR does not need to be mentioned
  at release time.
+1. The PR title is the **release note** you want published at release time.
+  * NOTE: PR titles are mutable and should reflect a release note friendly
+    message for any `release-note-*` labeled PRs.
+
+The only exception to these rules is when a PR is not a cherry-pick and is
```

Comment:
https://github.com/kubernetes/contrib/pull/738/files#diff-7294a858e5f70293980043d7fe211686R64 is a munger that should enforce this (as best we can).....